src/rust/bitbox02/src/event.rs

```diff
@@ -0,0 +1,7 @@
+pub use bitbox02_sys::event_slider_data_t;
+pub use bitbox02_sys::event_t;
+pub use bitbox02_sys::event_types;
+
+pub fn emit_event(event: *const event_t) {
```

Comment:
This is not a safe wrapper. You should be able to use `event: &event_t` instead or not?

Reply:
fixed